assets/src/components/account/GettingStartedOverview.tsx

```diff
@@ -2,32 +2,37 @@ import React from 'react';
 import {Box} from 'theme-ui';
 import * as API from '../../api';
 import Title from 'antd/lib/typography/Title';
-import {Paragraph, Input, colors, Text} from '../common';
+import {Button, Paragraph, Input, colors, Text} from '../common';
 import {TwitterPicker} from 'react-color';
 import SyntaxHighlighter from 'react-syntax-highlighter';
 import ChatWidget from '@papercups-io/chat-widget';
 import {atomOneLight} from 'react-syntax-highlighter/dist/esm/styles/hljs';
 
 type Props = {};
 type State = {
-  account: any;
+  account_id: any;
   color: string;
   title: string;
   subtitle: string;
+  id?: string;
 };
 
 class GettingStartedOverview extends React.Component<Props, State> {
   state: State = {
-    account: null,
+    account_id: null,
     color: colors.primary,
     title: 'Welcome to Papercups!',
     subtitle: 'Ask us anything in the chat window below 😊',
+    // company_name: null,
   };
 
   async componentDidMount() {
     const account = await API.fetchAccountInfo();
 
-    this.setState({account});
+    const {color, title, subtitle, id} = account.widget_config;
```

Comment:
this seems to break if no widget_config exists
<img width="1212" alt="Screen Shot 2020-07-26 at 2 57 38 PM" src="https://user-images.githubusercontent.com/5264279/88487136-842b8780-cf50-11ea-923c-cb0982e4af39.png">